internal/cluster/validator.go

```diff
@@ -528,67 +528,15 @@ func (v *clusterValidator) printNtpServerConfigured(c *clusterPreprocessContext,
 }
 
 func (v *clusterValidator) isOcsRequirementsSatisfied(c *clusterPreprocessContext) validationStatus {
```

Comment:
it will be generic operator method?

Reply:
i mean , i will glad to have as isOlmOpetatorsRequirementsSatisfied that will call some function that will verify which operators should be validate and validate them

Reply:
That's the idea  - it will be implemented in this way in the upcoming PRs